dockers/docker-fpm-frr/bgpd.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+addr="127.0.0.1"
+port=2601
+
+while getopts ":A:P:" opt; do
+    case "${opt}" in
+        A) addr=${OPTARG}
+            ;;
+        P) port=${OPTARG}
+            ;;
+    esac
+done
+
```

Comment:
you need shift $((OPTIND-1)

Reply:
Thanks for this comment, added the shift.